src/libs/sys-util-rs/src/mount.rs

```diff
@@ -450,7 +450,7 @@ fn compact_lowerdir_option(opts: &[String]) -> (String, Vec<String>) {
 
     let idx = idx as usize;
     let common_dir = get_longest_common_prefix(&lower_opts);
-    if common_dir == "/" {
```

Comment:
Please fold this clean up into the original patch for easier review.

Reply:
fixed